<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Logins" failures="1" tests="2" name="Default test" time="17.605" errors="0" timestamp="2021-08-23T11:41:25 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.208">
    <failure type="java.lang.AssertionError" message="expected [15] but found [12]">
      <![CDATA[java.lang.AssertionError: expected [15] but found [12]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.productup.stepdefinitions.ReusableSteps.in_response_body_should_be(ReusableSteps.java:71)
at âœ½."total" in response body should be "15"(file:///C:/Users/Logins/eclipse-workspace/APIAutomationPOC/src/test/resources/Features/demo.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.009"/>
</testsuite> <!-- Default test -->
